### DISCUSSION TITLE
Support code search over all orgs of an enterprise on github.com

### DISCUSSION THREAD

**Discussion by ehartung** [community/community#10223] [Enterprise]
As an enterprise cloud customer I'd love to be able to search code over all repositories in all organizations of my enterprise to which I have at least read access.

**Comment by look** [comment]
Thanks for the feedback @ehartung. Do you have ideas for how you'd like to see that supported in the UI or query language?

**Reply by ehartung** [reply]
Hi @look! I could imagine it as an additional scope. So in the UI that could be an additional item in the dropdown and for the query language maybe something like `ent: my-enterprise`. Does this make sense?

**Reply by bpan** [reply]
Why would this need to be an additional scope? Shouldn't this have been the default from the start?

When I search in my IDE it defaults to my current project (or maybe my project and libraries I pulled in). I almost never want to search across everything in the world first. In an enterprise, I'm most concerned about code or issues in my orgs and I'd want my search to work that way.

Being in GitHub should provide an advantage (I can use a familiar workflow and sometimes I can use the same search to find something more broadly), not be something to workaround (every search is a needle in a 14 million result haystack).

**Comment by devarshgandhi** [comment]
Another enterprise customer looking for a similar functionality to search for a code within GHEC enterprise account.

**Comment by jtgrohn** [comment]
I would also love to see this. Currently having to maintain a custom scope for this.
